### PR TITLE
974 - Content updates on Pricing and Guidance pages

### PR DESCRIPTION
**Pull request by jonathanbobel**
## Description

Small content updates to the [Pricing](https://docs.google.com/document/d/1MAr2oc3-D0FREYk8FPueWHHrSGJHHqopJcspzJ_zUws/edit#heading=h.a8r0tv111sgf) and [Guidance](https://docs.google.com/document/d/1HVUepE--JufiImezrc24o9qdErNiFDNHYifyfnSaIzY/edit) pages

(#974 )

### CODE REVIEW

**Review by stvnrlly — COMMENTED** [review]
👍🏻 on implementation, a couple specific questions for @maomeara63

File: app/templates/views/guidance/index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h2 class="padding-top-1" id="format-content">Format your content</h2>
   <h3>To create and format your message</h3>
   <ol class="list">
     <li>All messages start from a template</li>
-    <li>Click “<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>”. You’ll see existing templates.</li>
```
Comment (terrazoon) [inline app/templates/views/guidance/index.html:44]: heheh smart quotes.

File: app/templates/views/guidance/index.html [hunk 2]
```diff
@@ -84,6 +84,9 @@ <h4>Example</h4>
   <p>To personalize with the recipient's first name and include a reference number:</p>
   <p class="padding-2 bg-base-lightest">State WIC: Hello ((first name)), your reference is ((ref number)). Please provide this number when you call 123-123-1234 to make an appointment.</p>
```
Comment (terrazoon) [inline app/templates/views/guidance/index.html:85]: I don't know if 123 is a legit area code, but should we do '555' to guarantee the invalidity of the number?

Comment (terrazoon) [reply app/templates/views/guidance/index.html:85]: 123-555-1234 or 123-555-5555 or 555-555-5555 etc.

File: app/templates/views/guidance/index.html [hunk 3]
```diff
@@ -41,7 +41,7 @@ <h2 class="padding-top-1" id="format-content">Format your content</h2>
   <h3>To create and format your message</h3>
   <ol class="list">
     <li>All messages start from a template</li>
-    <li>Click “<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>”. You’ll see existing templates.</li>
+    <li>Click “<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>”. You'll see existing templates.</li>
```
Comment (ccostino) [inline app/templates/views/guidance/index.html:44]: Ah, the smart quotes persist here!  Let's get those fixed:

```suggestion
    <li>Click "<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>". You'll see existing templates.</li>
```

File: app/templates/views/guidance/index.html (hunk 2 above)
Comment (ccostino) [inline app/templates/views/guidance/index.html:85]: I think we can just make a suggestion here even though this isn't changed code.  Let's try!

```suggestion
  <p class="padding-2 bg-base-lightest">State WIC: Hello ((first name)), your reference is ((ref number)). Please provide this number when you call 555-555-5555 to make an appointment.</p>
```

**Review by ccostino — CHANGES_REQUESTED** [review]
Thanks, @jonathanbobel!

Let's remove the remaining smart quotes and see if we can fix the phone number quick while we're at it - hopefully the suggestions make it simple to do right here!

**Review by ccostino — APPROVED** [review]
Thanks, @jonathanbobel!

**Pull request merged**